brickstrap.sh

```diff
@@ -416,6 +455,7 @@ case "${cmd}" in
         create-rootfs
         create-tar
         create-image
+        dump-info-hook
```

Comment:
This will make `dump-info-hook` run twice since it is called in `run-hooks` as well.